src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -17,6 +17,7 @@
     public static final double DRIVETRAIN_FRONT_RIGHT_ENCODER_OFFSET = -Math.toRadians(221.924);
     public static final double DRIVETRAIN_BACK_LEFT_ENCODER_OFFSET = -Math.toRadians(214.980);
     public static final double DRIVETRAIN_BACK_RIGHT_ENCODER_OFFSET = -Math.toRadians(168.398);
+    public static final String DRIVETRAIN_INTAKE_CAN_BUS_NAME = "DrivebaseIntake";
```

Comment:
I am wondering if "Canivore" or something else is a better name.  That would cut down on confusion if we end up shifting more to the canivore.